src/components/DlDialogBox/DlDialogBox.vue

```diff
@@ -52,9 +62,12 @@
 <script lang="ts">
 import { v4 } from 'uuid'
 import { defineComponent, PropType } from 'vue-demi'
+import DlIcon from '../DlIcon.vue'
+import _ from 'lodash'
```

Comment:
Please dont import all of lodash here. use specific functions needed
ex: `import { throttle } from 'lodash'`